### PR TITLE
Support multiple fallback extensions

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | #233

### CODE REVIEW

File: src/ViewTrait.php [hunk 1]
```diff
@@ -204,17 +207,17 @@ public function getBasePath(): string
...
     }
 
     /**
      * Gets the fallback view file extension.
      *
-     * @return string The fallback view file extension.
+     * @return string[] The fallback view file extension.
      */
-    public function getFallbackExtension(): string
+    public function getFallbackExtension(): array
```
[the remark this answers is not in the document]
Comment (rustamwin) [reply src/ViewTrait.php:218]: We can just rename since this method is not released yet.

[the remark this answers is not in the document]
Comment (samdark) [reply src/ViewTrait.php:218]: Even better, yes!

**Review by samdark — APPROVED** [review]
👍 Need CHANGELOG adjusted to reflect the state of things now.

File: src/ViewTrait.php [hunk 2]
```diff
@@ -627,7 +630,16 @@ private function findTemplateFile(string $view): string
...
+        foreach ($this->fallbackExtensions as $fallbackExtension) {
+            $fileWithFallbackExtension = $file . '.' . $fallbackExtension;
+            if (is_file($fileWithFallbackExtension)) {
+                return $fileWithFallbackExtension;
+            }
+        }
+
+        [$firstFallbackExtension] = $this->fallbackExtensions;
+
+        return $file . '.' . $firstFallbackExtension;
```
Comment (vjik) [inline src/ViewTrait.php:642]:
```suggestion
        return $file . '.' . $this->fallbackExtensions[0];
```

**Review by vjik — APPROVED** [review]

**Pull request merged**